internal/service/ocm/ocm_impl.go

```diff
@@ -167,3 +191,13 @@ func createQueryOrder(q models.UserQuery) string {
 
 	return order
 }
+
+func createV3QueryOrder(q models.UserV3Query) string {
+	order := "organization.id"
```

Comment:
What would you think about moving this to a const right above the method? That way we're not allocing it every time the method is called.

Reply:
Fixed!!